parsebib.el

```diff
@@ -355,13 +386,38 @@ simply replace the match) or a function (the match will be
 replaced by the result of calling the function on the match
 string).  Earlier elements are evaluated before later ones, so if
 one string is a subpattern of another, the second must appear
-later (e.g. \"''\" is before \"'\").")
+later (e.g. \"\" is before \"'\").")
```

Comment:
Probably an accident: the two single quotes should not be deleted.

Reply:
It was, now fixed.